AttitudeManager/Inc/AM.hpp

```diff
@@ -12,19 +12,31 @@
 #include "AM_ControlAlgorithm.hpp"
 #include "CommonDataTypes.hpp"
 
+#include "FreeRTOS.h"
```

Comment:
Does `"FreeRTOS.h"` this need to be included in this file?

Reply:
Yes, in `semphr.h`:
![Screenshot 2023-07-24 at 9 05 01 PM](https://github.com/UWARG/ZeroPilot-SW-3.5/assets/70457170/c40ba6c0-acae-4835-a4b1-73439c2edaae)